Source/Charts/Data/Implementations/Standard/ChartDataSet.swift

```diff
@@ -26,24 +26,24 @@ open class ChartDataSet: ChartBaseDataSet
 {
     public required init()
     {
+        values = [ChartDataEntry]()
+
         super.init()
-        
-        _values = [ChartDataEntry]()
     }
     
     public override init(label: String?)
     {
+        values = [ChartDataEntry]()
+
         super.init(label: label)
-        
-        _values = [ChartDataEntry]()
     }
     
     @objc public init(values: [ChartDataEntry]?, label: String?)
     {
+        self.values = values ?? []
```

Comment:
if `[]` and `[ChartDataEntry]()` are both valid, shall we just use one of them?

Reply:
OK

Reply:
Just curious, do you know whose performance is better? like `[]` may need to check the type and call the initializer?

Reply:
The performance is identical in this case. This is a compile time check, not a runtime check. It just looks cleaner this way.